bin/rucio-admin

```diff
@@ -830,7 +831,19 @@ def declare_bad_file_replicas(args):
                     bad_files.append(bad_file)
     else:
         bad_files = args.listbadfiles
-    for chunk in chunks(bad_files, 500):
+
+    # Interpret filenames not in scheme://* format as LFNs and convert them to PFNs
+    bad_files_pfns = []
+    for bad_file in bad_files:
+        if bad_file.find('://') == -1:
+            scope, name = extract_scope(bad_file)
+            replicas = [replica for rep in client.list_replicas([{'scope': scope, 'name': name}])
+                        for replica in rep['pfns'].keys()]
+            bad_files_pfns.extend(replicas)
+        else:
+            bad_files_pfns.append(bad_file)
+
+    for chunk in chunks(bad_files_pfns, 500):
```

Comment:
There might be a problem in case the PFNs have different schemes

Reply:
OK - should I force one scheme in particular e.g. 'srm'?
Or I could skip all replicas that don't have the same scheme as the first item in the list.

Reply:
Now the list of PFNs is split into separate sublists - one sublist for each scheme